code/modules/events/electrical_storm.dm

```diff
@@ -42,7 +58,7 @@
 		var/obj/machinery/power/shield_generator/shield_gen = pick(shields)
 		//Minor breaches aren't enough to let through frying amounts of power
 		if(shield_gen.take_shield_damage(30 * severity, SHIELD_DAMTYPE_EM) <= SHIELD_BREACHED_MINOR)
-			return
+			shielded = TRUE
```

Comment:
Shouldn't it break here?

Reply:
Lack of break is intentional, just so rumbly storm sounds happen and such.

Reply:
Oh, thanks Github, it looked like this was in the for loop so it was hitting every shield.